src/runtime_src/core/tools/xbutil2/OO_Performance.cpp

```diff
@@ -77,16 +77,16 @@ OO_Performance::execute(const SubCmdOptions& _options) const
 
   try {
     if(boost::iequals(m_action, "DEFAULT")) {
```

Comment:
I see half this file uses `if (` while other half uses `if(`.  Consistency and `if (` is best.

Reply:
got it, I'll keep that in mind